core/src/test/scala/c4s/process/ProcessSpec.scala

```diff
@@ -58,6 +58,18 @@ class ProcessSpec extends Specification {
       }.unsafeRunSync()
     }
 
+    "run command reading a stream from another command" >> {
+      withProcess { implicit shell =>
+        createTmpDirectory[IO].use { path =>
+          for {
+            _ <- Process.runInPath("touch test-file", path)
+            result <- Process.runInPath("ls", path)
+            resultStream <- Process.run("wc", result.output)
+          } yield resultStream.exitCode == (ExitCode.Success)
```

Comment:
Maybe, we can verify the output of `wc`, just to verify it is consuming the input.